src/en/models-config.md

```diff
@@ -1,39 +1,44 @@
 Title: General configuration options  
-TODO: check options haven't changed
+TODO: Check accuracy of key table
+	Confirm 'all' harvest mode state. Seems it should be "'Dead' or
+	'Unknown'" OR "a combination of modes 'destroyed' and 'unknown'".
 
 
 # Configuring models
 
-There may be times, particularly when dealing with complex cloud environments
-or running multiple instances of Juju, where it would be useful to control
-further aspects of the models you create and the machines they deploy.
+A model influences all the machines that Juju creates within it and, in turn, the
+services that get deployed onto those machines. It is therefore a very powerful
+feature to be able to configure at the model level.
```

Comment:
I don't like the wording of this, but i can't really pinpoint why. Hmm. Never mind